main.js

```diff
@@ -83,7 +80,7 @@ if(!getVar(VARIABLE_FINISHED_SETUP, false)) run("Startup/Setup.js");
 
 if(!getVar(VARIABLE_FINISHED_FIRST_SESSION, false)) run("Session/FirstSession.js");
 
-
+showImage("Images/Spicy/Intro/SpicyGif*.gif", 7);
```

Comment:
Why show it again? Startup has  worked for me within the given duration. This way you  delay the startup by 7 seconds?

Reply:
good point.. I'll delete that line